src/d3a/models/market/one_sided.py

```diff
@@ -101,7 +101,8 @@ def get_offers(self) -> Dict:
     def offer(self, price: float, energy: float, seller: str, seller_origin,
               offer_id=None, original_price=None, dispatch_event=True,
               adapt_price_with_fees=True, add_to_history=True, seller_origin_id=None,
-              seller_id=None, attributes: Dict = None, requirements: List[Dict] = None) -> Offer:
+              seller_id=None, attributes: Dict = None, requirements: List[Dict] = None,
+              time_slot: DateTime = None) -> Offer:
```

Comment:
Nit, the same for the other arguments here and in other changed functions
```suggestion
              time_slot: Optional[DateTime] = None) -> Offer:
```